static/js/admin.js

```diff
@@ -287,6 +295,20 @@ function _setup_page() {
         error: failed_listing_streams
     });
 
+    // Populate realm aliases
+    channel.get({
+        url: "/json/realm/aliases",
+        idempotent: true,
+        timeout: 10*1000,
+        success: function (aliases_data) {
+            update_alias_label();
+            _.each(aliases_data.aliases, function (alias) {
+                add_alias(alias);
+            });
+        },
+        error: failed_listing_aliases
+    });
```

Comment:
I needed to make this request since currently `page_params.domains` contains only `alias.domain`. To avoid this request I would need to modify the `page_params.domains` to include `alias.id` as well(I need the alias' id because the a `DELETE` request's URI should uniquely identify the resource according to the REST API rules).